docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -13,46 +13,77 @@ image::images/lens_end_to_end_dashboard.png[Final dashboard vis]
 [[add-the-data-and-create-the-dashboard]]
 === Add the data and create the dashboard
 
-To create visualizations of the data from the web server, add the data set, then create the dashboard.
+If you are working with your own data, you should already have an <<index-patterns, index pattern>>.
+To install the sample logs that contain fake website traffic:
 
 . From the {kib} *Home* page, click *Try our sample data*.
 
 . From *Sample web logs*, click *Add data*.
 
+Then create a new dashboard:
+
 . Open the main menu, click *Dashboard*.
 
 . Click *Create dashboard*.
 
+. Set the <<set-time-filter,time filter>> to *Last 90 days*.
+
+You don't need to save the dashboard while it's in draft mode.
+
 [float]
 [[open-and-set-up-lens]]
 === Open and set up Lens
 
+*Lens* is the visualization builder recommended for most users.
 With *Lens*, you identify the data fields you want to visualize, drag and drop the fields, then watch as
 *Lens* uses heuristics to apply the fields and create a visualization for you.
 
-. From the dashboard, click *Create panel*.
-
-. On the *New visualization* window, click *Lens*.
+. From the dashboard, click *Create visualization*.
 +
 [role="screenshot"]
-image::images/lens_end_to_end_1_1.png[New visualization popover]
+image::images/lens_end_to_end_1_1.png[Dashboard create visualization button]
 
-. Make sure the *kibana_sample_data_logs* index appears.
+. Make sure the *kibana_sample_data_logs* index appears. You may need to select
+a different index pattern from the menu:
++
+[role="screenshot"]
+image::images/lens_end_to_end_1_2.png[Lens index pattern selector]
 
 [discrete]
 [[view-the-number-of-website-visitors]]
-=== View the number of website visitors
+=== Build your first visualization
 
-To determine how many users have visited your website within the last 90 days, create a metric visualization, then add it to the dashboard.
+Every time you build a visualization in *Lens*, you will follow these steps:
 
-. Set the <<set-time-filter,time filter>> to *Last 90 days*.
+. Do you know which chart type you'd like to use? If you do, select the type before
+dragging any fields. If you don't, you can change the chart types after configuring your functions.
+
+. Do you know exactly which functions you want? If so, click the dimension and
+configure it manually.
+
+. Do you want to use a specific field in a chart, but aren't sure which function? Drag the field
+to a specific dimension and Lens will pick a function for you.
```

Comment:
How about something like 'Drag the field to one of the groups in the config panel'?